cli/read.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Northern.tech AS
+// Copyright 2024 Northern.tech AS
```

Comment:
For future reference. We don't need to modify the copyright year anymore in the source files - we removed that check some time ago.

I remember from your WIP PR that you were having CI failures, but AFAIK this should not be the culprit.

Reply:
Thanks for the confirmation - I'm happy to hear that the simplification is valid here as well! When I got the CI failures related to this I wasn't sure, but I guess I started from a stale repo state and that's the culprit 😬.